samples_project/Assets/SampleViewer/Samples/StreamLayer/StreamLayerWebSocketSubscribe.cs

```diff
@@ -278,10 +278,10 @@ private void TryParseFeedAndUpdatePlane(string data)
         }
     }
 
-    // Populates the stadium drown down with all the stadium names from the Stadiums list
+    // Populates the flight drown down with all the flight names from the flights list
```

Comment:
I guess the original code for this part comes from another sample and the comments were not updated... (same for one below)